app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/presentation/character/CharacterProfileActivity.kt

```diff
@@ -0,0 +1,56 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.presentation.character
+
+import android.content.Context
+import android.os.Bundle
+import android.view.MenuItem
+import com.naxtlevelofandroiddevelopment.marvelgallery.R
+import com.naxtlevelofandroiddevelopment.marvelgallery.model.MarvelCharacter
+import com.naxtlevelofandroiddevelopment.marvelgallery.presentation.common.*
+import kotlinx.android.synthetic.main.activity_character_profile.*
+
+class CharacterProfileActivity : PresenterBaseActivity(), CharacterProfileView {
```

Comment:
sure but you don't even need to have the word Presenter in there. IMO BaseActivitity in your app may deal with Presenter and/or other aspects of the app without having to explicitly mentioned them in the name of the base activity.